main/autogen/autogen.h

```diff
@@ -82,6 +83,7 @@ struct ParsedFile {
     std::string toString(core::Context ctx);
     std::string toMsgpack(core::Context ctx, int version);
     void classlist(core::Context ctx, std::vector<std::string> &out);
+    void subclasses(core::Context ctx, std::map<std::string, std::set<std::string>> &out);
```

Comment:
Can this(and the method above) rather than using out-parameters return their results?
Also, can they be given names that are "actions" rather than "nouns"?